os.kieserver.launch/added/kieserver-config.sh

```diff
@@ -393,6 +393,8 @@ function setKieServerEnv() {
     fi
     KIE_SERVER_OPTS="${KIE_SERVER_OPTS} -Dorg.kie.server.repo=${KIE_SERVER_REPO}"
     KIE_SERVER_OPTS="${KIE_SERVER_OPTS} -Dorg.optaplanner.server.ext.disabled=${KIE_SERVER_BRP_DISABLED}"
+    KIE_SERVER_OPTS="${KIE_SERVER_OPTS} -Dorg.kie.server.mgmt.api.disabled=${KIE_SERVER_MGMT_DISABLED}"
+    KIE_SERVER_OPTS="${KIE_SERVER_OPTS} -Dorg.kie.server.startup.strategy=${KIE_SERVER_STARTUP_STRATEGY}"
```

Comment:
Can the above two new variables please be echoed out in the dumpKieServerEnv() function?